components/mpas-seaice/cime_config/buildnml

```diff
@@ -801,12 +801,7 @@ def buildnml(case, caseroot, compname):
             lines.append('    <var name="iceVolumeTendencyTransport"/>')
             lines.append('    <var name="iceAreaTendencyThermodynamics"/>')
             lines.append('    <var name="iceAreaTendencyTransport"/>')
-            lines.append('    <var name="iceAgeTendencyThermodynamics"/>')
-            lines.append('    <var name="iceAgeTendencyTransport"/>')
-            lines.append('    <var name="iceAgeCell"/>')
             lines.append('    <var name="firstYearIceAreaCell"/>')
```

Comment:
Please also remove firstYearIceAreaCell here too.

Reply:
This has been removed, along with firstYearIceAreaCell from BGC output.